### PR TITLE
fix(semrel): correctly set version tags

### PR DESCRIPTION
**Pull request by c0rydoras**

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -44,6 +44,10 @@ jobs:
           images: ghcr.io/adfinis/timed-backend
           flavor: |
             latest=auto
+          tags: |
+            type=semver,pattern={{version}},value=${{ steps.semrel.outputs.version }}
+            type=semver,pattern={{major}}.{{minor}},value=${{ steps.semrel.outputs.version }}
+            type=semver,pattern={{major}},value=${{ steps.semrel.outputs.version }}
```
Comment (hairmare) [inline .github/workflows/release.yaml:50]: don't we also need {{major}}.{{minor}}.{{patch}}?

Comment (c0rydoras) [reply .github/workflows/release.yaml:50]: thats what `version` is for AFAIK

Comment (c0rydoras) [reply .github/workflows/release.yaml:50]: image
https://github.com/docker/metadata-action?tab=readme-ov-file#typesemver

Comment (hairmare) [reply .github/workflows/release.yaml:50]: tru dat

**Review by hairmare — APPROVED** [review]

**Pull request merged**